tests/integration-tests/tests/createami/test_createami/test_build_image/image.config.yaml

```diff
@@ -23,3 +23,7 @@ DeploymentSettings:
         - {{ private_subnet_id }}
         SecurityGroupIds:
         - {{ default_vpc_security_group_id }}
+
+DevSettings:
+    Cookbook:
+        ChefCookbook: https://github.com/himani2411/aws-parallelcluster-cookbook/tarball/wip/rocky/
```

Comment:
we shouldn't point to your remote, let's use develop instead

Reply:
Ohh I made these changes for testing. I forgot I had a PR open